src/dmd/dinterpret.d

```diff
@@ -47,6 +47,21 @@ import dmd.tokens;
 import dmd.utf;
 import dmd.visitor;
 
+
+void incUsageCtfe(InterState* istate, const ref Loc loc)
+{
+    if (global.params.cov && istate)
```

Comment:
Is `istate == null` ever valid? The callers of `incUsageCtfe` use `istate` without explicit `null` checks AFAICT.

Reply:
it seems to be. I wouldn't have added the check if it were unnecessary.